src/models/generated/HydroReservoir.jl

```diff
@@ -0,0 +1,160 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
+    mutable struct HydroReservoir <: DeviceParameter
+        name::String
+        available::Bool
+        area::Area
+        topology::Union{Nothing, HydroTopology}
+        storage_volume_limits::MinMax
+        spillage_outflow_limits::Union{Nothing, MinMax}
+        volume_to_power_factor::Float64
```

Comment:
What's the difference between this and the conversion factor? The power conversion is for turbines no?

Reply:
I removed the repeated field.